src/TextArea/TextArea.tsx

```diff
@@ -251,9 +259,11 @@ export class TextArea extends React.Component<Props, State> {
         )}
       >
         <div className="TextArea--infoRow">
-          <label className="TextArea--label" htmlFor={this.props.name}>
-            {this.props.label}
-          </label>
+          {this.props.label && (
```

Comment:
I wonder how many TextArea's exist today with neither a `label` prop specified nor a manually configured label (like in the MessagingInput). It's probably very few. But worth noting that those may end up completely label-less after this is rolled out and imported in repos.

One alternative would be to expose a `noLabel` prop, though I'm not a huge fan. I might just do a quick pass through launchpad's TextArea's to make sure all specify a `label` prop. I know FamPo is okay.

Reply:
Okay LOL, there are quite a few in launchpad. Literally the first two that I looked at haha.

- https://github.com/Clever/launchpad/blob/03be51877180c72625a97be086e7ec39ab9fbd46/src/ui/components/ClassroomActivityExperiment/ClassroomActivityExperiment.tsx#L90-L100
- https://github.com/Clever/launchpad/blob/03be51877180c72625a97be086e7ec39ab9fbd46/src/ui/components/CollectionPageHeader/EditTeacherAnnouncementModal.tsx#L80-L89

Reply:
Hmm, I think I'm not understanding how this change makes those cases any worse. It is true that there may now be some cases where the `<label>` is no longer included, but is there any benefit to rendering a label with no text? It's not clear to me what makes that preferable over no label.

Reply:
Ah of course 🤦‍♂️

You're absolutely right. Missed the piece that we're just rendering an empty label for those today.

Ignore me :hide:

Reply:
Haha no worries, glad we're on the same page 😄